### PR TITLE
chore: correct stage panel groupings

### PR DESCRIPTION
**Pull request by gabemontero**
stage-top

stage-bottom

@amisstea fingers crossed PTAL .... I am trying to sanity check as well

Signed-off-by: Gabe Montero <gmontero@redhat.com>

rh-pre-commit.version: 2.3.0
rh-pre-commit.check-secrets: ENABLED

### CODE REVIEW

File: dashboards/grafana-dashboard-rhtap-slos.configmap.yaml [hunk 1]
```diff
@@ -602,36 +407,49 @@ data:
...
           "options": {
             "code": {
               "language": "plaintext",
               "showLineNumbers": false,
               "showMiniMap": false
             },
-            "content": "<h1><center>95% of pipeline run duration excludes execution overhead</center></h1>",
-            "mode": "html"
+            "content": "<h1><center>90% of the Releases should be validated below 5 seconds</center></h1>",
+            "mode": "markdown"
```
Comment (gabemontero) [inline dashboards/grafana-dashboard-rhtap-slos.configmap.yaml:452]: this feels wrong, no @amisstea ?

File: dashboards/grafana-dashboard-rhtap-slos.configmap.yaml [hunk 2]
```diff
@@ -1066,333 +968,387 @@ data:
...
       "tags": [],
       "templating": {
         "list": [
           {
             "current": {
-              "selected": true,
+              "selected": false,
               "text": "rhtap-observatorium-production",
-              "value": "rhtap-observatorium-production"
+              "value": "P22466E8E7855F1E0"
```
Comment (gabemontero) [inline dashboards/grafana-dashboard-rhtap-slos.configmap.yaml:1351]: this also feels wrong, doesn't it @amisstea

Comment (amisstea) [inline dashboards/grafana-dashboard-rhtap-slos.configmap.yaml:1351]: Please restore these settings. Thanks!

File: dashboards/grafana-dashboard-rhtap-slos.configmap.yaml [hunk 3]
```diff
@@ -1066,333 +968,387 @@ data:
...
+            "reduceOptions": {
+              "calcs": [],
+              "fields": "",
+              "values": false
             },
+            "showThresholdLabels": false,
+            "showThresholdMarkers": false,
+            "sizing": "auto"
+          },
+          "pluginVersion": "10.4.1",
```
Comment (gabemontero) [inline dashboards/grafana-dashboard-rhtap-slos.configmap.yaml:1207]: maybe the newer plugin version has bearing on some of my other comments ?

Comment (amisstea) [reply dashboards/grafana-dashboard-rhtap-slos.configmap.yaml:1207]: I'm not sure but this version looks unchanged here.

File: dashboards/grafana-dashboard-rhtap-slos.configmap.yaml (hunk 2 above)
Comment (amisstea) [reply dashboards/grafana-dashboard-rhtap-slos.configmap.yaml]: And the default selected option?

File: dashboards/grafana-dashboard-rhtap-slos.configmap.yaml (hunk 3 above)
Comment (gabemontero) [reply dashboards/grafana-dashboard-rhtap-slos.configmap.yaml:1207]: ack

File: dashboards/grafana-dashboard-rhtap-slos.configmap.yaml (hunk 2 above)
Comment (gabemontero) [reply dashboards/grafana-dashboard-rhtap-slos.configmap.yaml]: updated

File: dashboards/grafana-dashboard-rhtap-slos.configmap.yaml (hunk 1 above)
Comment (amisstea) [reply dashboards/grafana-dashboard-rhtap-slos.configmap.yaml]: It's probably ok since markdown can render html tags.

**Review by amisstea — APPROVED** [review]

**Pull request merged**